app/views/early_career_payments/claims/_ineligibility_reason_ineligible_nqt_in_academic_year_after_itt.html.erb

```diff
@@ -0,0 +1,15 @@
+<p class="govuk-body">
+  If you’re a trainee teacher and plan to complete your studies in this academic
+  year you might be able to apply in the next academic year.
+</p>
+
+<h3 class="govuk-heading-m">
```

Comment:
@martikat In the prototype this is an `h1` element. Where did the `h3` get specified. I have looked in both V10 and V11 documents and both have this as a `h1`.

Reply:
Updated this to be `h1`